packages/cerebral-provider-firebase/src/remove.js

```diff
@@ -1,13 +1,14 @@
 import {
   createRef
 } from './helpers'
+import {FirebaseProviderError} from './errors'
 
 export default function remove (path) {
   const ref = createRef(path)
-  return new Promise((resolve, reject) => {
-    ref.remove().then(
-      () => resolve({}),
-      (error) => reject({error: error.message})
-    )
-  })
+
+  return ref.remove()
+    .then(() => undefined)
```

Comment:
Damn ES6 arrow functions... so easy to read them wrong. I though you mean returning an object which would cause a merge. Just wanted to be explicit about returning `undefined`